cmd/storage.go

```diff
@@ -117,13 +118,38 @@ var storageListCmd = &cobra.Command{
 		storageList, err := storage.List(client, componentName, applicationName)
 		checkError(err, "Failed to list storage")
 
-		if len(storageList) == 0 {
+		hasMounted := false
+		hasUnmounted := false
+		w := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+		w1 := tabwriter.NewWriter(os.Stdout, 5, 2, 3, ' ', tabwriter.TabIndent)
+
+		for _, storage := range storageList {
+			if storage.Path != "" {
+				if !hasMounted {
+					hasMounted = true
+					fmt.Fprintln(w, "NAME", "\t", "SIZE", "\t", "PATH")
+				}
+				fmt.Fprintln(w, storage.Name, "\t", storage.Size, "\t", storage.Path)
+			} else {
+				if !hasUnmounted {
+					hasUnmounted = true
+					fmt.Fprintln(w1, "NAME", "\t", "SIZE")
+				}
+				fmt.Fprintln(w1, storage.Name, "\t", storage.Size)
+			}
+		}
+		if hasMounted {
+			fmt.Printf("The component '%v' has the following storage attached -\n", componentName)
+			w.Flush()
+		} else {
 			fmt.Printf("The component '%v' has no storage attached\n", componentName)
+		}
+
```

Comment:
It would be nice to have one empty line between mounted and unmounted section. It will make a nice visual division.

Reply:
@kadel You mean a dashed line division?

Reply:
no just empty line.
Instead of this:
```
▶ odo storage list
The component 'frontend' has the following storage attached -
NAME     SIZE     PATH
data     100M     /data
The following unmounted storages can be mounted to 'frontend' -
NAME              SIZE
data-frontend     100M
```

it should look like this:
```
▶ odo storage list
The component 'frontend' has the following storage attached -
NAME     SIZE     PATH
data     100M     /data

The following unmounted storages can be mounted to 'frontend' -
NAME              SIZE
data-frontend     100M
```